lib/collectionspace/converter/publicart/config.yml

```diff
@@ -128,7 +128,7 @@ registered_profiles:
           authority_subtype: person
         - name_field: movementcontactshared
           authority_type: Person
-          authority_subtype: person
+          authority_subtype: person_shared
```

Comment:
@Paulmulonzia - I merged in your config updates and corrected them here. I won't repeat what I said on the org and person PRs you've put in. :-)